README.md

```diff
@@ -0,0 +1,83 @@
+## The Example App end-to-end tests
+
+These e2e tests are used to ensure that all implementations of `The Example App` share the same functionality. You may use this repository as reference how to implement end-to-end tests via [cypress](https://www.cypress.io/) but it is not needed at all to get `The Example App` running on your machine nor to write a project using Contentful.
```

Comment:
all web-based implementations

Reply:
Split the last sentence in two and also make sure that you are consistent with saying "end-to-end" tests. remove "e2e" 

"You may use this repository as a reference for implementing end-to-end tests with [cypress](https://www.cypress.io/). It is not necessary to use this repo to get the example app running on your machine.